<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>unknownsettingstab</class>
 <widget class="QWidget" name="unknownsettingstab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_unknown_settings">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>50</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Unknown or custom settings:</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>701</width>
     <height>551</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_add_setting">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>700</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_remove_setting">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>700</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
